dts/riscv32/rv32m1.dtsi

```diff
@@ -289,17 +289,17 @@
 			#address-cells = <1>;
 			#size-cells = <1>;
 
-			m4_flash: flash@0 {
+			flash0: flash@0 {
```

Comment:
Why are you changing this? Even though we're not using the cortex m cores, the hardware reference manual refers to these regions as m4 and m0 flashes.

Reply:
I removed it temporarily. And then I will try to raise a requirement to update the RM to make the flash region name easy to understand.

Reply:
That was my doing @MaureenHelm . In my version of this change I didn't adopt the cortex naming and discussed with Lyle about changing the name.